src/main/java/org/traccar/notificators/NotificatorPushover.java

```diff
@@ -74,23 +77,26 @@ public void sendSync(long userId, Event event, Position position) {
             return;
         }
 
-        Message message = new Message();
```

Comment:
I think it's better to rename `org.traccar.notification.Message` to something like `NotificationMessage`. That way we don't need to rename everything else.